ci/azure-travis-template.yaml

```diff
@@ -67,7 +67,7 @@ steps:
     pip install --upgrade ${{parameters.numpy_spec}} &&
     pip install --upgrade pip setuptools==59.6.0 wheel build meson meson-python &&
     pip install ${{parameters.other_spec}}
-    "cython==0.29.33"
+    "git+https://github.com/matusvalo/cython.git@struct_visibility"
```

Comment:
There's a lot of work going on at the moment transferring Azure jobs to GHA. The best place to test changes for these cython 3 changes is https://github.com/scipy/scipy/blob/main/.github/workflows/linux_meson.yml#L293.

i.e. remove the pinned cython install on that line and either install cython on the  `python -m pip install --pre --upgrade` line, or use this specific line on L295.